doc/checkpkg.1.asciidoc

```diff
@@ -19,6 +19,19 @@ outputs if there are soname differences for the new package. A directory is
 also created using mktemp with files containing a file list for both packages
 and a library list for both packages.
 
+Options
+-------
+
+*-r, --rmdir*::
+	Remove the temporary directory created to contain the file and library list
+	of both packages.
+
+*-w, --warn*::
+	Pring a warning instead of a regular message in case of soname differences.
```

Comment:
I don't recognize the word "pring", what is it a typo of? :p

Reply:
lol, most likely "print". Maybe I know English too well to be able to proofread this??? Thanks, @Foxboron.